src/AzureStack/CommerceAdmin/Commerce.Admin/generate.cmd

```diff
@@ -4,6 +4,7 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/commerce latest Azure current azure-rest-api-specs %CD%
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/commerce latest Azure bcb638a668dbb79eaf3190324ca6d690be84b9f6 azure-rest-api-specs %CD%
```

Comment:
Please create a `generate.ps1` file like [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.
This should also generate/modify a .txt file, please commit those changes

Reply:
Done.